### PR TITLE
Read all files / write all files 

### PR DESCRIPTION
**Pull request by SciLor**
I have created a parameter to read and write all files from the flash for backup purposes.

(I have also added the latest changes from the original repo)

PS: could you please enable issues?

### CODE REVIEW

**Comment by mon** [comment]
Thanks for the heads up, enabled issues.

Having to confirm writes is not really consistent with how all the other commands work. Could you modify it so it instead writes by default, and shows the files that will be written with something like a `--dry-run` flag?

**Comment by SciLor** [comment]
added --simulate flag and removed --write

File: cc3200tool/cc.py [hunk 1]
```diff
@@ -1103,6 +1205,13 @@ def main():
         if command.cmd == "list_filesystem":
             cc.list_filesystem(command.json_output, command.inactive)
 
+        if command.cmd == "read_all_files":
+            cc.read_all_files(command.local_dir)
+
+        if command.cmd == "write_all_files":
+            cc.write_all_files(command.local_dir, command.write)
```
Comment (mon) [inline cc3200tool/cc.py]: Should this now be `command.simulate`?

Comment (SciLor) [reply cc3200tool/cc.py]: yes should be, fixed

**Comment by mon** [comment]
Thanks!

**Pull request merged**